src/tests/Directory.Build.props

```diff
@@ -194,5 +194,13 @@
     <TestFramework>GeneratedRunner</TestFramework>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- On Linux arm coreclr, the updated tracing/eventpipe/common/Microsoft.Diagnostics.NETCore.Client/Microsoft.Diagnostics.NETCore.Client.csproj project
+         does not yet work for several eventpipe runtime tests. We segment those tests with the following property and symbol -->
+    <RuntimeFlavor Condition="'$(RuntimeFlavor)' == ''">coreclr</RuntimeFlavor>
```

Comment:
I would move this up to line 73 near the other one.

Reply:
Done, thanks!